packages/foreman/nodejs-history/nodejs-history.spec

```diff
@@ -16,38 +19,49 @@ Source5: https://registry.npmjs.org/value-equal/-/value-equal-0.4.0.tgz
 Source6: https://registry.npmjs.org/resolve-pathname/-/resolve-pathname-2.2.0.tgz
 Source7: https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.13.2.tgz
 Source8: https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz
-Source9: %{name}-%{version}-registry.npmjs.org.tgz
-BuildRequires: nodejs-packaging
```

Comment:
We rely on this at least for `%nodejs_sitelib` in non-SCL builds. In SCL this is provided by `rh-nodejs10-build`.